### PR TITLE
Fix mac errors for local server

### PR DESCRIPTION
**Pull request by RichieEscarez**
Disable live reload for the Hugo server by default (`LIVERELOAD=" `[`--watch=false --disableLiveReload`](https://gohugo.io/commands/hugo_server/#options)`"`). Docs get copied and fixed by the script so live reload is not useful for building only the docs. This also fixes the "too many files open" error in Macs when running the Hugo server.

Add option to enable live reload (ie for when modifying the site itself): `localbuild.sh -s reload` 
For Macs, this will cause the "too many files open" error.

Updated the instructions.

### CODE REVIEW

File: scripts/localbuild.sh [hunk 1]
```diff
@@ -52,6 +52,11 @@ BUILDALLRELEASES="false"
 BUILDSINGLEBRANCH="false"
 PRBUILD="false"
 
+# Default Hugo build options
+# disable Hugo server
+SERVER=""
+# disable live reload
+LIVERELOAD=" --watch=false --disableLiveReload"
```
Comment (RichieEscarez) [inline scripts/localbuild.sh:59]: primary change (1/3) - by default disable Hugo server live reload

File: scripts/localbuild.sh [hunk 2]
```diff
@@ -109,20 +125,24 @@ while getopts f:b:a:s arg; do
...
+      # See scripts/docs-version-settings.sh for the list of the built branches.
       BUILDALLRELEASES="${OPTARG}"
       BUILDENVIRONMENT="production"
       BUILDSINGLEBRANCH="false"
       ;;
     s)
-      echo 'Running hugo in server mode'
+      echo 'Running Hugo server'
       SERVER="server"
+      if [ "${OPTARG}" = "reload" ]; then
```
Comment (RichieEscarez) [inline scripts/localbuild.sh:141]: primary change (2/3) -  if you use the "reload" flag, enable live reload in Hugo server

File: scripts/localbuild.sh [hunk 3]
```diff
@@ -134,7 +154,7 @@ source scripts/processsourcefiles.sh
 
 # BUILD MARKDOWN
 # Start HUGO build
-hugo $SERVER --baseURL "" --environment "$BUILDENVIRONMENT"
+hugo $SERVER --baseURL "" --environment "$BUILDENVIRONMENT" $LIVERELOAD --gc
```
Comment (RichieEscarez) [inline scripts/localbuild.sh:157]: primary change (3/3) -  also added `--gc` to clean cache each time (ie. using live reload is useful only for site updates) https://gohugo.io/commands/hugo_server/

**Comment by RichieEscarez** [comment]
@evankanderson or @mattmoor 

If one of you get a free moment can you PTAL and help me merge this change? It's been working flawlessly for me over the last couple of weeks.

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

Sorry, I looked at this earlier but apparently forgot to comment.

**Pull request merged**